mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule BroadwayCloudPubSub.MixProject do
     [
       {:broadway, "~> 0.3.0"},
       {:google_api_pub_sub, "~> 0.4"},
+      {:tesla, "~> 1.2.0"},
+      {:hackney, "~> 1.6"},
```

Comment:
I'm ok with making hackney required to keep it simple,  we now can additionally simplify installation guide: https://github.com/plataformatec/broadway_cloud_pub_sub/blob/master/README.md#installation

If someone would rather use a different adapter I think we can revisit this in the future and drop the dependency on hackney but still make it de-facto required as in people would have to add it to their deps list.